### PR TITLE
refactor: remove invalid textAlign property in View component

### PR DESCRIPTION
**Pull request by jarvisluong**
* remove invalid textAlign property in View component

| Question         | Response    |
| ---------------- | ----------- |
| Version?         | v1.4.1      |
| Devices tested?  | iPhone 6, 7, 8, X |
| Bug fix?         | no      |
| New feature?     | no      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |

---

## Description

I removed textAlign property in View component style since it is not a valid style property and cause a warning in the console

### CODE REVIEW

**Comment by machour** [comment]
Hi @jarvisluong and thank you!
Since this change have to deal with UI, could you provide us with before/after screenshots?

Thank you!

**Comment by jarvisluong** [comment]
@machour this PR does not change the UI at all 😄 I was just triggered by the warning in the console that complains about invalid style attribute 😄

**Comment by machour** [comment]
I know, but I still need to make sure that "it doesn't break anything" 😝 

So either you provide the screenshots and I can look at them and validate, or I'll have to try your PR locally and it will take a bit more time. Both way works for me 😄

**Comment by jarvisluong** [comment]
@machour  So here comes the screenshots 😄 

## Screenshots



| Before   | After    |
| -------- | -------- |
|before|after|

File: .all-contributorsrc [hunk 1]
```diff
@@ -712,6 +712,15 @@
         "code",
         "design"
       ]
+    },
+    {
+      "login": "jarvisluong",
+      "name": "Luong Dang Hai",
```
Comment (housseindjirdeh) [inline .all-contributorsrc]: 🚀

**Pull request merged**